pkg/clustermesh/clustermesh.go

```diff
@@ -207,12 +208,13 @@ func (cm *ClusterMesh) Close() {
 
 func (cm *ClusterMesh) newRemoteCluster(name, path string) *remoteCluster {
 	rc := &remoteCluster{
-		name:        name,
-		configPath:  path,
-		mesh:        cm,
-		changed:     make(chan bool, configNotificationsChannelSize),
-		controllers: controller.NewManager(),
-		swg:         lock.NewStoppableWaitGroup(),
+		name:           name,
+		configPath:     path,
+		mesh:           cm,
+		changed:        make(chan bool, configNotificationsChannelSize),
+		controllers:    controller.NewManager(),
+		swg:            lock.NewStoppableWaitGroup(),
+		releaseTimeout: 5 * time.Second,
```

Comment:
Why are we setting it to 5 seconds? Can you leave a comment on where this value came from?